Slim/Plugin/Deezer/strings.txt

```diff
@@ -106,32 +106,39 @@ PLUGIN_DEEZER_SKIPS_EXCEEDED
 
 PLUGIN_DEEZER_SMART_RADIO
 	EN	Deezer Smart Radio
+	FR	Deezer Smart Radio
```

Comment:
When a translation is identical to the English, then it's not strictly necessary to provide a translation, as LMS would fall back to the English version. Just a way to save a few seconds 😉 .

Reply:
Thank you for the info. I remove the line.
But, for the next time, what do you prefer?
(It doesn't take a long time to add an identical string, and if it saves a few milliseconds maybe it's better?)

Reply:
As you like. I don't really care. I'm only lazy, thus usually don't add it unless needed.